triage/experiments/base.py

```diff
@@ -30,13 +34,16 @@ class ExperimentBase(object):
     """The Base class for all Experiments."""
     __metaclass__ = ABCMeta
 
+    cleanup_timeout = 60  # seconds
```

Comment:
I guessed that a minute was a reasonable length of time after which to give up on the state table clean-up task; but, certainly, happy to change this to something more reasonable.

Reply:
yeah, I doubt cleaning up the state table should take more than a minute in any reasonable case, so that seems fine to me.